<launch>

    <!-- params to compare with /scout_odom -->
    <param name="initial_x" value="0"/>
    <param name="initial_y" value="0"/>
    <param name="initial_theta" value="0"/>

    <node pkg="rosbag" type="play" name="bag1" args="/array/first_project/src/bags/bag1.bag"/>

    <node pkg="project" type="params" name="params"/>
    <node pkg="project" type="pub_twist" name="pub_twist" output="screen"/>
    <node pkg="project" type="pub_odom" name="pub_odom" output="screen"/>
    <node pkg="project" type="tf_node" name="tf_node"/>

    <node pkg="tf2_ros" type="static_transform_publisher" name="world"
          args="-0.598297894001 0.442829757929 0.330364078283
           -0.00782093405724 0.0201139133424 0.64408993721 0.764645338058 world odom"/>

</launch>